mappings/net/minecraft/world/biome/BiomeAccess.mapping

```diff
@@ -1,6 +1,6 @@
 CLASS net/minecraft/class_4543 net/minecraft/world/biome/BiomeAccess
 	CLASS class_4544 Storage
-		METHOD method_16359 getStoredBiome (III)Lnet/minecraft/class_1959;
+		METHOD method_16359 getBiome (III)Lnet/minecraft/class_1959;
```

Comment:
May be confusing with `world.getBiome` as this xyz is not block xyz, neither are they chunk xyz

Reply:
It's basically the same as the 1.14 `getBiome` method but 3d and not using block xyz

Reply:
we have another one that use blockpos, people will get confused man

Reply:
but why "get **Stored** biome?"

Reply:
maybe sampleBiome would be better

Reply:
because the actual getbiome method goes through the access strategy to get the correct biome for the block

and this 3 int method is like getting the raw biome stored, so it is not sampling either but rather getting the stored biome in the 3d array with the 3d index

Reply:
Oh! so this is kinda like the getBiomeForNoiseGen method from 1.14 but slightly different

Reply:
actually if someone can find a way to optimize the get biome call through the biome strategy, they may be able to improve mc performance as currently many mob ai and block logic call getbiome and everytime the biome is get the game goes through all the computations within the biome access types